src/test/functional/functional-tests.js

```diff
@@ -622,27 +629,35 @@ describe('functional tests', function() {
       client.removeObject(bucketName, _5mbObjectName, done)
     })
   })
-
+  let policyNotImplemented = false
   describe('bucket policy', () => {
     let policies = [Policy.READONLY, Policy.WRITEONLY, Policy.READWRITE]
 
     // Iterate through the basic policies ensuring it can set and check each of them.
     policies.forEach(policy => {
+      policyNotImplemented = false
       step(`setBucketPolicy(bucketName, objectPrefix, bucketPolicy, cb)_bucketPolicy:${policy}_`, done => {
         client.setBucketPolicy(bucketName, '', policy, err => {
-          if (err) return done(err)
+          if (err) {
+            if(err.code != 'NotImplemented') done(err)
```

Comment:
Please make sure `status = 'NA'` and `alert = 'Not Implemented'` without having any error message in `message` and any stack trace in `error` fields in the logs.

Reply:
There won't be any error and stack messages. Currently, there is no way to pass status 'NA' to the repoter. Will file a separate issue for it and work on it there.